dex/models/_projects/oneinch/arbitrum/oneinch_arbitrum_ar_raw_traces.sql

```diff
@@ -0,0 +1,24 @@
+{% set blockchain = 'arbitrum' %}
+
+
+
+{{
+    config(
+        schema = 'oneinch_' + blockchain,
+        alias = 'ar_raw_traces',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_hash', 'trace_address']
```

Comment:
another thing we can consider for performance gains:
can we add `block_date` to unique keys any time it's a partition column? that'll help trino reduce partitions to search for hash values on merge join conditions

Reply:
done